packages/rollup-plugin-html/test/fixtures/assets/styles-with-referenced-assets.css

```diff
@@ -0,0 +1,15 @@
+#a1 {
+  background-image: url('image-a.png');
+}
+
+#a2 {
+  background-image: url('image-a.svg');
+}
+
+#d1 {
+  background-image: url('./image-d.png');
+}
+
+#d2 {
+  background-image: url('./image-d.svg');
```

Comment:
some deduplication of images was happening, I think because we try to identify images with same hash and merge them into one, so it's a feature of this plugin
but it made testing the exclusion cumbersome, so I added a new `image-d.svg/png` with the content equal to the file name to get a different hash than for `image-a` and see in the output all expected exclusions

the deduplication continues to work correctly, it's just that test would be hard to read if I mix so much different functionality in one place